rotkehlchen/api/rest.py

```diff
@@ -1405,8 +1405,7 @@ def delete_custom_asset(self, identifier: str) -> Response:
                 # Before deleting, also make sure we have up to date global DB owned data
                 self.rotkehlchen.data.db.update_owned_assets_in_globaldb(cursor)
                 self.rotkehlchen.data.db.delete_asset_identifier(cursor, identifier)
-                with GlobalDBHandler().conn.write_ctx() as cursor:
-                    GlobalDBHandler().delete_custom_asset(cursor, identifier)
+                GlobalDBHandler().delete_asset_by_identifier(cursor, identifier)
```

Comment:
This is wrong, you are using the cursor for the user db in the globaldb. This is the reason for one of the tests to fail but there seems to be more errors